mps-model-adapters-plugin/src/test/kotlin/org/modelix/model/mpsadapters/ReplaceNodeTest.kt

```diff
@@ -30,10 +31,12 @@ class ReplaceNodeTest : MpsAdaptersTestBase("SimpleProject") {
 
         val repositoryNode: INode = MPSRepositoryAsNode(mpsProject.repository)
 
-        writeActionOnEdt {
+        runCommandOnEDT {
             val module = repositoryNode.getChildren(BuiltinLanguages.MPSRepositoryConcepts.Repository.modules)
                 .single { it.getPropertyValue(BuiltinLanguages.jetbrains_mps_lang_core.INamedConcept.name) == "Solution1" }
-            val model = module.getChildren(BuiltinLanguages.MPSRepositoryConcepts.Module.models).single()
+            val model = module.getChildren(BuiltinLanguages.MPSRepositoryConcepts.Module.models)
+                .single { (it as MPSModelAsNode).model !is GenericDescriptorModelProvider.DescriptorModel }
+
```

Comment:
In this test I get the error:

```
Couldn't add it to:  EnumClass "Class1"[3688054287334907424] in <no model>
jetbrains.mps.smodel.IllegalModelAccessException: [member] InstanceMethodDeclaration "method1"[3688054287334907470] in Solution1.model1 already has parent: [root] ClassConcept "Class1"[3688054287334907424] in Solution1.model1
Couldn't add it to:  EnumClass "Class1"[3688054287334907424] in <no model>
	at jetbrains.mps.smodel.SNode.insertChildBefore(SNode.java:801)
	at jetbrains.mps.smodel.SNode.addChild(SNode.java:840)
	at org.modelix.model.mpsadapters.MPSNode.replaceNode(MPSNode.kt:87)
```

(One also needs to change `ConceptReference("mps:f30...")` to `ConceptReference("f30...")`)